src/main/java/com/devonfw/ide/sonarqube/common/impl/SonarDevon4jPlugin.java

```diff
@@ -1,30 +1,94 @@
 package com.devonfw.ide.sonarqube.common.impl;
-
 import org.sonar.api.Plugin;
 import org.sonar.api.PropertyType;
 import org.sonar.api.config.PropertyDefinition;
-
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
 /**
  * The {@link Plugin} to integrate devonfw architecture rules into SonarQube.
  */
 public class SonarDevon4jPlugin implements Plugin {
-
   static final String CONFIG_KEY = "sonar.devon.config";
-
   static final String FORBIDDEN_CONF_KEY = "sonar.devon.forbiddenConf";
-
   static final String DISABLED = "Disabled";
-
   static final String ISSUES_SEVERITY_KEY = "sonar.Devon.preview.issuesSeverity";
-
+  private static final String QUALINSIGHT = "qualinsight-plugins-sonarqube-smell-plugin";
+  private static final String PMD = "sonar-pmd-plugin";
```

Comment:
Please make sure to use proper devonfw formatting. You can either do it manually by copying the format of code that is already on the master branch, or simply import devonfw format settings into your IDE.